AirCasting/Map/LocationTracker.swift

```diff
@@ -13,13 +13,18 @@ class LocationTracker: NSObject, ObservableObject, CLLocationManagerDelegate {
     let locationManager: CLLocationManager
     @Published var locationGranted: LocationState
     @Published var allLocations: [CLLocation]
+    @Published var googleLocation: [PathPoint]
     
     init(locationManager: CLLocationManager) {
         self.locationManager = locationManager
         self.locationManager.desiredAccuracy = kCLLocationAccuracyBest
         switch locationManager.authorizationStatus {
-            case .authorizedAlways, .authorizedWhenInUse: self.locationGranted = .granted
-            case .denied, .notDetermined, .restricted:  self.locationGranted = .denied
+            case .authorizedAlways, .authorizedWhenInUse:
+                self.locationGranted = .granted
+                googleLocation = [PathPoint(location: CLLocationCoordinate2D(latitude: (locationManager.location?.coordinate.latitude)!, longitude: (locationManager.location?.coordinate.longitude)!), measurement: 20)]
```

Comment:
we 100% sure force unwrapping location here is safe?

Reply:
I think yes, because it will be only triggered when ` case .authorizedAlways, .authorizedWhenInUse:` so current location should be available then

Reply:
+1 to what Paweł wrote, I'm not a fan of force unwrapping so are 100% sure? :D